src/parser.js

```diff
@@ -54,6 +55,14 @@ const parser = yargs
       default: false,
       type: 'boolean',
     },
+    prettier: {
+      describe: oneLine`
+          Prettier configuration options 
+          to be passed to prettier-eslint
+          using dot-notation`,
+      type: 'string',
```

Comment:
I went searching way too far in yargs to be sure to grasp how it works !

The different types are `string` `array` `number` `boolean` `count`

`array` is for having different values in a flag map to an array: 
For example `--prettier a b c` would return `[a, b, c]` 

Concluded none of them is really appropriate here, so maybe we can just omit the type ! 

However, I noticed a cryptic error if the flag wasn't given keys and values:
`--prettier a b c` or `--prettier` would return `Invalid value used as weak map key` 
Tracing back the error, I noticed it was due to the use of the coerce function. 

I am writing a quick fix in the coerce function to check if the dot-notation is correct, if not, it would throw something like `chalk.red(You should use dot-notation with the --prettier flag, for example, --prettier.singleQuote)`